sr_multi_moveit/ur10srh_moveit_config/scenes/constrained_space.scene

```diff
@@ -0,0 +1,23 @@
+Workspace
+* wall_front
```

Comment:
will need to modify those to be more sensible once the robot is mounted  but it's an example